### DISCUSSION TITLE
✅ Project testing / Pruebas del proyecto ✅

### DISCUSSION THREAD

**Discussion by Omitg24** [Arquisoft/lomap_es3a#39] [Ideas]
<p align="center">
<img src="https://user-images.githubusercontent.com/91057639/218811134-2eec75dc-a6d5-42f7-b1d6-fae6bd5ac158.png" width="640" height="320">
</p>

# EN: Discussion objective 
<p align="justify">
Here we will talk about the testing of the project, showing what we would like to do in order to test the project we are developing, we will also offer help when any team member needs it.
</p>

# ES: Objetivo de la discusión
<p align="justify">
Aquí hablaremos sobre las pruebas del proyecto, mostrando lo que nos gustaría hacer para probar el proyecto que estamos desarrollando, también ofreceremos ayuda cuando algún miembro del equipo lo necesite.
</p>

**Comment by raulEspaa** [comment]
# Pruebas Usabilidad
Buenas equipo! Como veríais comenzar a testear con usuarios el prototipo que tenemos, para ver que opinan de la interfaz.

Un saludo, Raúl

**Reply by Omitg24** [reply]
Me parece buena idea, no obstante, en lugar de hacerlas ya, esperaría a tener algo más funcionando, ya sea la carga de marcadores, ventanas de Ayuda y Acerca de... y similar, para poder realizar unas pruebas en condiciones.

Un saludo,
Omar

**Reply by Omitg24** [reply]
## Comenzar con las pruebas
<p align="justify">
Hola equipo!
<br>
Tal y como comentó @UO278036 hace un mes, igual deberíamos ir empezando a realizar las pruebas, por lo que, que tal veis, dejar esta semana para reparar al máximo todos los bugs posibles, y para la próxima semana, ¿empezar las pruebas con una primera tanda?
</p>

<p align="justify">
Si tenéis alguna idea de alguna prueba que os gustaría incluir dejadlo por aquí, ya que, yo había pensado en hacer un máximo de 3 pruebas (de un tamaño considerable), pero si alguno tenéis una idea o algo que queráis realizar por mi genial.
</p>

<p align="justify">
Cabe mencionar, que había pensado en hacer unas 3 rondas de unas 4 personas, si vemos que podemos ampliar y realizar más rondas, debido a que vamos bien de tiempo y tenemos gente suficiente, podríamos ampliar, pero eso como véais.
</p>

Un saludo,
Omar
